.github/workflows/lint.yaml

```diff
@@ -14,3 +14,24 @@ jobs:
         uses: frenck/action-yamllint@v1
         with:
           path: catalog/
+  tknparse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # ∈(2,0) where 0 = all history, and 2 is the minimum.
```

Comment:
why do we need all history? Doesn't only HEAD matter?

Reply:
Probably we dont need and Indeed just HEAD matter. It was a copy+paste from the another action - I will fix it.